test/Formatter/Psr3FormatterTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Horde Log package
+ *
+ *
+ * @author     Laurenz Gass <gass@b1-systems.de>
+ * @category   Horde
+ * @license    http://www.horde.org/licenses/bsd BSD
+ * @package    Log
+ * @subpackage UnitTests
+ */
+
+namespace Horde\Log\Test;
+
+use PHPUnit\Framework\TestCase;
+
+use Horde\Log\Formatter\Psr3Formatter;
+use Horde\Log\LogLevel;
+use Horde\Log\LogMessage;
+
+use TypeError;
+
+class Psr3FormatterTest extends TestCase
+{
+    public function SetUp(): void
+    {
+        $this->messageLog = new LogLevel(3, 'info');
+        $this->message = new LogMessage($this->messageLog, '{datum} everything not saved, will be lost {name}');
+    }
+
+    public function testFormatterReplacesMessageWithContext()
+    {
+        $this->formatter = new Psr3Formatter(['name' => 'Voldemort','datum' => '1550071894']);
```

Comment:
you dont need to save the formatter as a class attribute here.
Also applies to other lines. Not a problem, but not needed.